shell/browser/electron_browser_main_parts.cc

```diff
@@ -270,7 +270,7 @@ void ElectronBrowserMainParts::PostEarlyInitialization() {
 
   v8::HandleScope scope(js_env_->isolate());
 
-  node_bindings_->Initialize();
+  node_bindings_->Initialize(js_env_->context());
```

Comment:
```suggestion
  node_bindings_->Initialize(js_env_->isolate()->GetCurrentContext());
```

Reply:
`js_env_->context()` returns `isolate_->GetCurrentContext()` - is there some potential edge case there i'm not seeing?

Reply:
Nope, the helper `js_env_->context` was removed in https://github.com/electron/electron/commit/478ce969143cd8d069ab4def0992e8d5719445cf#diff-dd5319dd51edbe558e1f667c5cce519d2c2cecaa23e6dc28f488216e9669f4e9L39 . So if we bring back the helper then would be consistent to make changes other previous call sites as well. On the contrary, if we don't want the helper we can instead use `js_env_->isolate()->GetCurrentContext()`

Reply:
sounds fine - i'll update to just call `js_env_->isolate()->GetCurrentContext()`